plugin/group_replication/CMakeLists.txt

```diff
@@ -113,6 +113,14 @@ ADD_DEFINITIONS(-DMYSQL_SERVER)
 ADD_DEFINITIONS(-DLOG_SUBSYSTEM_TAG="Repl")
 ADD_DEFINITIONS(-DLOG_COMPONENT_TAG="group_replication")
 
+IF(GROUP_REPLICATION_WITH_ROCKSDB)
```

Comment:
Why do we need introduce a new cmake option `GROUP_REPLICATION_WITH_ROCKSDB`?
Shouldn't
```
      if (trans_param->tables_info[table].db_type != DB_TYPE_INNODB &&
          trans_param->tables_info[table].db_type != DB_TYPE_ROCKSDB) {
```
always be enabled when we compile with RocksDB?

A bit rephrasing, is having RocksDB enabled and deliberately not wanting group replication plagin to have support for it a valid use case?

Reply:
Yes @percona-ysorokin  MyRocks DOES NOT WORK with GR and we do not want anyone in the general public to accidentally use it this way.  This option is to allow us to build it in a way that one can enable it for the purposes of testing.

Reply:
Oh. Now it makes sense. Thanks @georgelorchpercona